run.sh

```diff
@@ -14,7 +14,9 @@ socok8s_absolute_dir="$( cd "$(dirname "$0")" ; pwd -P )"
 DEPLOYMENT_MECHANISM=${DEPLOYMENT_MECHANISM:-"kvm"}
 
 # The base directory where workspace(s) are created in
-SOCOK8S_WORKSPACE_BASEDIR=${SOCOK8S_WORKSPACE_BASEDIR:-~}
+SOCOK8S_WORKSPACE_BASEDIR_DEFAULT=${TMPDIR:-${XDG_CACHE_HOME:-$HOME/.cache}}
```

Comment:
The artifacts in the workspace includes generated passwords. They need to be kept around for the user to find the passwords to use for some services, for example, shipyard service account, or try to connect to MariaDB directly on cli, or looked up during the upgrade/redeploy so we don't generate new passwords. Default to user home is not ideal, maybe /opt/soc or /opt/airship?

Reply:
Cache is indeed incorrect. ~/.config/socok8s would be another choice. @dirkmueller what can we use to find that home config folder? It's not in env vars IIRC (not a data folder)

Reply:
@evrardjp @jgu17 thanks for the explanations! I have fixed this.